LIBSRC/SMB2_SHC/COM/smb2_shc.c

```diff
@@ -370,6 +400,59 @@ int32 __MAPILIB SMB2SHC_GetVoltLevel(enum SHC_PWR_MON_ID pwr_mon_nr, u_int16 *vo
 	return SMB2_SHC_ERR_NO;
 }
 
+/****************************************************************************/
+/** Set power cycle duration in milliseconds
+ *
+ *  \param     status           \IN  desired delay in milliseconds
+ *  \return    SMB2_SHC_ERR_NO on success or error code
+ *
+ *  \sa SMB2SHC_SetPowerCycleDuration
```

Comment:
\sa generates a "see also" link in doxygen docu. Therefore, the SetXxx function should refer to the counterpart GetXxx function. 
\sa SMB2SHC_SetPowerCycleDuration -> \sa SMB2SHC_GetPowerCycleDuration